### DISCUSSION TITLE
Default setting and option changes

### DISCUSSION THREAD

**Discussion by stax76** [staxrip/staxrip#587] [General]
This discussion can be used to suggest default setting and options changes.

Two things I would like:

- A setting I always use is 'Save projects automatically', I think modern apps usually have such options enabled by default, VS Code and Sublime Text for instance have auto save enabled by default.
- As theme Dark Blue, or in general a dark theme.

**Comment by ghost** [comment]
In addition, I think UI scaling should be defaulted to 1.20. EDIT : I see @Dendraspis confused... so forget about it 🤣 . But think that most people have FHD and 4K displays.... And that everything (web sites, etc) have now a bigger look....

**Reply by Dendraspis** [reply]
Even bigger??
I also use StaxRip on a 1080p display and StaxRip is almost too big.  😂   I struggle to have Processing and Jobs window opened side by side.  😕 

> And that everything (web sites, etc) have now a bigger look....

Yeah, you have to zoom out to be able to read it .. and in 2020 the scroll wheel was more often used than between 1990 and 2010!!

**Reply by ghost** [reply]
🙄 well... are you using a Windows scaling setting that increases everything ?... 
Here if I leave staxrip to ui scale 1, it's unreadable. When I use ui scale =1.25 then I can put Processings and Jobs side by side easily...

**Reply by Dendraspis** [reply]
Nope, 100% and UIScale 1.00.

**Reply by stax76** [reply]
> In addition, I think UI scaling should be defaulted to 1.20.

You could increase your DPI, I'm using 288 DPI (300%) with a 27" 4K monitor.

> I struggle to have Processing and Jobs window opened side by side. 😕

They both are resizable and remember their size. 😀

**Reply by Dendraspis** [reply]
> They both are resizable and remember their size. 😀

I know, but I want them both on screen and Jobs with almost full displayed path without clicking and moving around.  😎

**Comment by Dendraspis** [comment]
> A setting I always use is 'Save projects automatically', I think modern apps usually have such options enabled by default, VS Code and Sublime Text for instance have auto save enabled by default.

Yeah, but only for settings, not projects, source files or similar. Sometimes changes are made to test something, so if you don't pay attention it could mess your projects up. I get this message quite often and I click almost everytime on *Don't save*.  😁 

But to be honest:  Doesn't bother me a lot. Whenever I start with fresh settings, I have to check all settings and invert the *danger zone*. So clicking one checkbox more or less is completely no matter as long it is optional.  😁 

***BUT*** what concerns me more is that you wanted to take a break from StaxRip, also because you don't use it anymore for months or even years.  So what's going on here?!?  😁

**Reply by ghost** [reply]
well I agree that the autosave feature will not be suitable for some people (including me), so if you decide to make it ON by default, i'll remove in my setup. not a problem...

as for the other concern expressed by @Dendraspis I undertsand he is very confused (don't get me wrong: we're happy to see you back)

**Reply by stax76** [reply]
I use Danger Zone only sometimes for development. You must have good memory if you don't edit version names.

Improving staxrip is kind of a habit..., also there is the TaskDialog 😉, but yes, I've big plans for mpv.net!

**Comment by curiosport** [comment]
So it has already been done?
`A setting I always use is 'Save projects automatically'`

I have my working environment ready and would like to just drag the video into StaxRip and have the .srip automatically saved and the job automatically added to the queue.

If there is an error it cancels the auto save.

Or is there any way to apply a specific setting/profile for all videos in a folder?
